sections/featured-collection.liquid

```diff
@@ -21,31 +21,49 @@
   }
 {%- endstyle -%}
 
+{% unless section.settings.show_description or section.settings.description != empty %}
+  {%- style -%}
+    .collection__title {
+      position: relative;
+      transform: translate(calc({{ section.settings.horizontal_offset }}px / 2), calc({{ section.settings.vertical_offset }}px / 2));
+      z-index: 1;
+    }
+
+    @media screen and (min-width: 750px) {
+      .collection__title {
+        transform: translate({{ section.settings.horizontal_offset }}px, {{ section.settings.vertical_offset }}px);
+      }
+    }
+  {%- endstyle -%}
+{%- endunless -%}
+
 {%- liquid
   assign products_to_display = section.settings.collection.all_products_count
 
   if section.settings.collection.all_products_count > section.settings.products_to_show
     assign products_to_display = section.settings.products_to_show
     assign more_in_collection = true
   endif
+
+  assign slider_is_enabled = false
+  if section.settings.enable_mobile_slider or section.settings.enable_desktop_slider
+    assign slider_is_enabled = true
+  endif
 -%}
 
 <div class="color-{{ section.settings.color_scheme }} isolate gradient">
-  <div class="collection page-width{% if section.settings.swipe_on_mobile == true and section.settings.collection.all_products_count > 2 and section.settings.products_to_show > 2 %} page-width-desktop{% endif %} section-{{ section.id }}-padding">
+  <div class="collection page-width section-{{ section.id }}-padding{% if section.settings.enable_mobile_slider == true and section.settings.collection.all_products_count > 2 and section.settings.products_to_show > 2 %} page-width-desktop{% endif %}">
     {% unless section.settings.title == blank %}
-      <div class="title-wrapper-with-link{% if section.settings.title == blank %} title-wrapper-with-link--no-heading{% endif %}{% if section.settings.collection.all_products_count > 2 and section.settings.swipe_on_mobile and section.settings.products_to_show > 2 %} title-wrapper--self-padded-tablet-down{% endif %} title-wrapper--no-top-margin">
-        <h2 class="title{% if section.settings.title == blank %} title--no-heading{% endif %}">{{ section.settings.title | escape }}</h2>
-  
-        {%- if section.settings.show_view_all and section.settings.swipe_on_mobile and more_in_collection -%}
-          <a href="{{ section.settings.collection.url }}" class="link underlined-link large-up-hide">{{ 'sections.featured_collection.view_all' | t }}</a>
-        {%- endif -%}
+      <div class="collection__title title-wrapper--no-top-margin {{ section.settings.heading_alignment }}{% if section.settings.collection.all_products_count > 2 and section.settings.enable_mobile_slider and section.settings.products_to_show > 2 %} title-wrapper--self-padded-tablet-down{% endif %}">
```

Comment:
@ludoboludo Is this common to use the reference `tablet-down` for device related items?  This is the first time I read about this. Just curious!

Reply:
I think it was specific here to deal with keeping the title with the right spacing when the slider is enabled on mobile but also on desktop. It's not super common but was done here for the specific use case.